edb/server/protocol/protocol.pyx

```diff
@@ -749,7 +749,7 @@ cdef class HttpProtocol:
                 path_parts[1:],
                 self.server,
                 self.tenant,
-                self.is_tenant_host,
+                is_tenant_host=not self.is_tls or self.is_tenant_host,
```

Comment:
I don't understand what this change is / why we're making it?

Reply:
> Also allows HTTP access on multitenat host system API.

This is mostly for the test who accesses the tenant-host system APIs (`/status/ready` and `/status/alive`) through cleartext HTTP, but also opens up access from any unencrypted clients. Without this change, such clients will get `No such tenant configured` from `/status/ready` or `/status/alive`.

Reply:
can we make this only work for the test then? allowing anyone to access metrics from any tenant without authentication is a minor security hole, but there's no reason to add it unnecessarily